_includes/landing.html

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
 
-    <p class="text-muted wow animated slideInUp" data-wow-delay=".15s">{{ site.description }}</p>
+    <p class="text-muted wow animated slideInUp" data-wow-delay=".15s">"{{ site.description }}"</p>
```

Comment:
I think this way we will be forcing using quotes around the description in the landing page. Some people may not like that behavior. Removing the quotes from the `_config.yaml` would be enough.